cargo/build.rs

```diff
@@ -24,7 +24,7 @@ fn generate_language_bindings() {
     }
 
     // TODO: Add support for print functions in a no_std environment
-    if cfg!(feature = "intel_64bit_systemv_nasm") {
+    if cfg!(not(feature = "std")) {
```

Comment:
If we take the suggestion to use a `enable_printers` feature, it would be used here to enable that feature in the code generator. This line would become:

```rust
if cfg!(not(feature = "enable_printers")) {
```